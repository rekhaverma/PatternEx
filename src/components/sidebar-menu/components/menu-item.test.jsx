import React from 'react';
import MenuItem from './menu-item.component';

describe('<MenuItem />', () => {
  let wrapper;
  let props;

  beforeEach(() => {
    props = {
      active: false,
      id: 'test',
      className: 'sidebarMenu__test',
      disabled: false,
      hasSubmenu: false,
      icon: 'test-icon',
      title: 'Test',
      onClick: jest.fn(),
    };
    wrapper = shallow(<MenuItem {...props} />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

