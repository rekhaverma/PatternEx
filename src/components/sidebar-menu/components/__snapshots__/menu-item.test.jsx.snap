// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuItem /> should render correctly 1`] = `
<div
  className="sidebarMenu__test"
  onClick={[Function]}
>
  <span
    className="test-icon"
  />
  <span>
    Test
  </span>
</div>
`;
